OpenBSD is missing the _Static_assert macro (should be in <assert.h>).

Index: src/openrct2/common.h
--- src/openrct2/common.h.orig
+++ src/openrct2/common.h
@@ -193,6 +193,7 @@ using rct_string_id = uint16_t;
 #    define RESTRICT __restrict__
 #endif
 
+#define static_assert _Static_assert
 #define assert_struct_size(x, y) static_assert(sizeof(x) == (y), "Improper struct size")
 
 #ifdef PLATFORM_X86
